public/app/plugins/panel/graph/graph.ts

```diff
@@ -709,6 +710,31 @@ class GraphElement {
     };
   }
 
+  getMonthDayFormat() {
+    if (contextSrv.user.monthDayFormat && contextSrv.user.monthDayFormat !== 'browser') {
+      return contextSrv.user.monthDayFormat;
+    }
+
+    if (!this.toLocaleDateStringSupportsLocales()) {
+      return '%m/%d';
+    }
+
+    const language = navigator.languages ? navigator.languages[0] : navigator.language || 'en-US';
+    return new Date(Date.UTC(1977, 11, 20, 0, 0, 0))
+      .toLocaleDateString(language, { month: 'numeric', day: 'numeric' })
+      .replace('12', '%m')
+      .replace('20', '%d');
+  }
+
+  toLocaleDateStringSupportsLocales() {
```

Comment:
could this be renamed to isLocaleDateStringLocalesSupported ,  reading this code I struggled to understand that this function was bool check and not a conversion function

Reply:
agree, canUse is better :+1: